plutus/budget_manager/project_budget.py

```diff
@@ -0,0 +1,65 @@
+from plutus.lib.constants import (
+    APP,
+    PLUTUS_CONFIG_TYPE_PROJECT,
+    PLUTUS_CONFIG_TYPE_PARENT,
+    PLUTUS_CONFIG_TYPE_LABEL,
+    PLUTUS_CONFIG_TYPE_DEFAULT
+)
+import logging
+import markus
+import sys
+
+metrics = markus.get_metrics(APP + ".plutus")
+
+
+class ProjectBudget:
+    '''
+    ProjectBudget class for representing a configuration object
+
+    Fields:
+    -------
+    project_id - str - GCP project id
+    budget_type - str - "AMT or LASTMONTH"
+    budget_amount - int - monthly budget amount in USD
+    products - list(str) - currently only ['ALL'] supported
+    alert_emails - list(str) - emails to alert when budget thresholds hit
+    threshold_rules - list(object) - objects have threshold_percent and spend_basis fields
+    include_credits - bool - Include GCP credits in the budget calculations
+    pubsub - bool - Send budget alerts to pubsub? Required for pagerduty/slack
+    pubsub_topic - str - Optional. Will default to default_pubsub_topic
+
+    billing_account_id - str - GCP billing account id
+    display_name - budget display name
+    config_type - str - The type of budget. Either project, parent, label, or default
+    parent_id - str - Optional - GCP parent folder id. For use with config_type parent
+    label_list - str - Optional - list of label k/v pairs. For use with config_type label
+
+    '''
+
+    def __init__(self, project_dict, config_type, billing_account_id, default_pubsub_topic):
+        self.logger = logging.getLogger(APP + '.projectbudget')
```

Comment:
nit, in-place string concatenation is not recommended. 

```suggestion
        self.logger = logging.getLogger(f"{APP}.projectbudget")
```